scripts/HPC_chatbot_preprocessor/parsed_mds/generic/FAQ/FAQ_paragraph_1.txt

```diff
@@ -0,0 +1,33 @@
+Frequently Asked Questions (FAQ)
```

Comment:
the FAQ needs to be treated separately: question and answer are one document. This intro block can be a separte block as well